grayskull/pypi/pypi.py

```diff
@@ -911,28 +945,30 @@ def _get_run_req_from_requires_dist(self, requires_dist: List) -> List:
         :return:
         """
         run_req = []
-        for req in requires_dist:
-            list_raw_requirements = req.split(";")
-            selector = ""
-            if len(list_raw_requirements) > 1:
-                list_extra = PyPi._get_extra_from_requires_dist(
-                    list_raw_requirements[1]
+        print(f"This is requires_dist: {requires_dist}")
+        if requires_dist:
```

Comment:
Can you explain what's going on here? The diff is too big due to the indentation change (which I was not expecting). 

I don't think we need the `if` clause here becuase if `requires_dist` is empty the `for` loop won't even start.